workers/pyproject.toml

```diff
@@ -50,7 +50,7 @@ pep8-naming = "^0.12.1"
 pre-commit = "^2.15.0"
 black = "^22.3.0"
 flake8 = "^4.0.1"
-isort = "^5.10.1"
+isort = "5.12.0"
```

Comment:
Is there a reason for not using `^5.12.0`?

Reply:
No. Thanks for spotting it. I corrected it.